openfisca_france/model/prestations/minima_sociaux/ass.py

```diff
@@ -146,3 +90,59 @@ def function(self, simulation, period):
         ass = ass * not_(ass < ass_params.montant_plein)  # pas d'ASS si montant mensuel < montant journalier de base
 
         return period, ass
+
+
+@reference_formula
+class ass_base_ressources_i(SimpleFormulaColumn):
+    column = FloatCol
+    label = u"Base de ressources individuelle de l'ASS"
+    entity_class = Individus
+
+    def function(self, simulation, period):
+        period = period.start.offset('first-of', 'month').period('month')
+        previous_year = period.start.period('year').offset(-1)
+
+        salaire_net = simulation.calculate_add('salaire_net', previous_year)
+        rstnet = simulation.calculate('rstnet', previous_year)
+        pensions_alimentaires_percues = simulation.calculate('pensions_alimentaires_percues', previous_year)
+        pensions_alimentaires_versees_individu = simulation.calculate('pensions_alimentaires_versees_individu', previous_year)
+
+        aah = simulation.calculate('aah', previous_year)
+        indemnites_stage = simulation.calculate('indemnites_stage', previous_year)
+        revenus_stage_formation_pro = simulation.calculate('revenus_stage_formation_pro', previous_year)
+
+        return period, salaire_net + rstnet + pensions_alimentaires_percues - abs_(pensions_alimentaires_versees_individu) + aah + indemnites_stage + revenus_stage_formation_pro
```

Comment:
Hi, 
`abs` is used in the same context in  `aspa.py` and `cmu.py`. 
Maybe tou want to correct it there too ;)